lib/parseItem.js

```diff
@@ -120,7 +122,7 @@ const parseVideo = obj => {
       author.navigationEndpoint.commandMetadata.webCommandMetadata.url;
   }
   const badges = Array.isArray(obj.badges) ? obj.badges.map(a => a.metadataBadgeRenderer.label) : [];
-  const isLive = badges.some(b => b === 'LIVE NOW');
+  const isLive = badges.some(b => b.match(/^(live|live now)$/i));
```

Comment:
Regex.test has better performance than string.match when doing a boolean check btw

Reply:
Yeah, figured that out soon after committing 😅